query-service-api/src/main/proto/request.proto

```diff
@@ -73,6 +74,12 @@ message ColumnIdentifier {
   string alias = 2;
 }
 
+message ObjectIdentifier {
```

Comment:
new message should use consistent snake_casing.

Also does this replace column identifier? Do we deprecate that?

Reply:
resolved